ci-operator/config/stackrox/acs-fleet-manager/stackrox-acs-fleet-manager-main.yaml

```diff
@@ -39,7 +39,7 @@ resources:
   '*':
     requests:
       cpu: 2000m
-      memory: 6000Mi
+      memory: 8000Mi
```

Comment:
Add a TODO with date applied, date you think it will no longer apply (or a release version that should go out when it can be reverted/revisited), and a tiny bit of context. Otherwise we'll never come back and request 8 GiB forever.